### PR TITLE
Laboratory work #8, Pareshina Evgenia - 20FPL2

### PR DESCRIPTION
**Pull request by eapareshina**

### CODE REVIEW

File: lab_7_llm/main.py [hunk 1]
```diff
@@ -2,28 +2,20 @@
 Neural machine translation module.
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: Checkout files of lab 7 to master

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+datasets==2.16.1
+--extra-index-url https://download.pytorch.org/whl/cpu
+pandas==2.1.4
+sentencepiece==0.2.0
+torch==2.2.0
+torchinfo==1.8.0
+transformers==4.37.2
```
Comment (SalnikovIgor) [inline requirements.txt]: New line

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (eapareshina) [reply requirements.txt]: done

File: lab_7_llm/main.py (hunk 1 above)
Comment (eapareshina) [reply lab_7_llm/main.py]: done

**Pull request closed without merging**